src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -1234,6 +1234,7 @@ default List<Member> getMembersByEffectiveName(@Nonnull String name, boolean ign
     @Nonnull
     default List<Member> getMembersWithRoles(@Nonnull Role... roles)
     {
+        Checks.noneNull(roles, "Roles");
```

Comment:
You actually don't need it since you can't pass `null` in here.

The compiler wouldn't know if `getMembersWithRoles(Role...)` or `getMembersWithRoles(List<Role>)` is meant so it won't compile. And if you pass nothing, it will be an empty array.

But I also had to think about that for a second. Because if it would be possible, it would be needed.